### DISCUSSION TITLE
[Documentation Request]: How/where do you link elements to thesauri?

### DISCUSSION THREAD

**Discussion by howaskew** [AstunTechnology/os-custom-geonetwork#12] [General]
If I add or update vocabularies, where and how do you adjust the settings for an element to allow a user to select from the new terms?

**Comment by archaeogeek** [comment]
@howaskew that depends where it should be used. If it's for standard keywords (like the existing thesauri) then once you add the vocabulary it should be present in the keyword picker, eg:
![Screenshot from 2021-02-02 11-26-46](https://user-images.githubusercontent.com/1544849/106594121-944ca100-6549-11eb-879d-89d2c6e6c49c.png)
If it's related to some other element then we may need to adjust the editor layout configuration.

**Comment by howaskew** [comment]
I'm thinking both I guess. Can cover when we get to adding other elements.

**Comment by archaeogeek** [comment]
Converted this to a discussion rather than an issue as it's a bit more general
